pygotham/users/models.py

```diff
@@ -20,6 +20,8 @@
     db.Column('role_id', db.Integer, db.ForeignKey('roles.id')),
 )
 
+user_talks = db.Table('speakers', Speaker.metadata, autoload=True)
```

Comment:
Per the docs (the code sample immediately before [this section](http://docs.sqlalchemy.org/en/latest/orm/basic_relationships.html#deleting-rows-from-the-many-to-many-table)), you can just use the table name here.